processor/processor.go

```diff
@@ -2441,7 +2446,7 @@ func throughputPerSecond(processedJob int, timeTaken time.Duration) int {
 }
 
 func (proc *HandleT) crashRecover() {
-	proc.gatewayDB.DeleteExecuting(jobsdb.GetQueryParamsT{CustomValFilters: []string{GWCustomVal}, JobCount: -1})
+	proc.gatewayDB.DeleteExecuting(GWCustomVal)
```

Comment:
Can't we have the custom Val empty here just like we do it everywhere else?

Reply:
Yes, if you believe this is safe, it would simplify even further the `DeleteExecuting` method!

Reply:
we can have the parameters in `DeleteExecuting` probably for any future use-cases but we don't need to do it at GW level since we have only one customVal